src/Base/RestAPI/ItemFields/Yoast.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace OWC\OpenPub\Base\RestAPI\ItemFields;
+
+use WP_Post;
+
+class Yoast extends SeoPress
```

Comment:
Ik zou Yoast geen child class maken van SeoPress. Yoast en Seopress zijn allebei "SEO plugins". 

Het zou logischer zijn als je een parent class hebt, bijvoorbeeld `abstract class SEO` waar zowel de `class Seopress` en `class Yoast` een child van zijn.

De gedeelde methods gaan dan in de abstracte parent class `SEO`.

Reply:
Check